scripts/lib/CIME/XML/env_mach_pes.py

```diff
@@ -17,6 +18,12 @@ def __init__(self, case_root=None, infile="env_mach_pes.xml", components=None):
         schema = os.path.join(get_cime_root(), "config", "xml_schemas", "env_mach_pes.xsd")
         EnvBase.__init__(self, case_root, infile, schema=schema)
 
+    def add_comment(self, comment):
+        if comment is not None:
+            tmp = GenericXML(root_name_override="comment", infile="fake", read_only=False)
```

Comment:
This is a bit awkward, but I couldn't find a cleaner way.

Reply:
@jedwards4b , I see now that make_child cannot support what you want since it attaches the child as a SubElement. Probably the easiest thing here would be:

```
comment = self.make_child("comment", text=comment)
self.remove_child(comment)
self.add_child(comment, position=1)
```